plugins/misc/projects/src/main/java/org/apache/hop/projects/gui/UsageDate.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.hop.projects.gui;
```

Comment:
missing header in file

Reply:
... because this is not used and I deleted it.  I pushed its removal.